DESCRIPTION

```diff
@@ -17,16 +17,18 @@ Authors@R:
              email = "f.kelpin@umcg.nl"))
 Depends:
     R (>= 3.6),
-    DSI,
+    DSI
+Imports:
     methods,
-    httr
+    httr,
+    stringr
```

Comment:
This took me a while.
When you install a package, all packages mentioned in Depends and Imports get installed.
But when you load it, only the ones in Depends get loaded.
This is good, cause you are then not polluting the user's search space with methods used in your dependencies.
And you can still use them as long as you reference them with namespace::method syntax.

Reply:
So, since our users will not be making calls to methods, stringr and httr package themselves, we should move them from Depends to Imports.